test/server/routes/applications.test.js

```diff
@@ -238,11 +238,11 @@ describe('routes/applications.js', () => {
       .set('Cookie', cookieStr)
       .send(routeData.updateApplication)
       .then(response => {
-        expect(stubLogger).to.have.callCount(5);
+        expect(stubLogger).to.have.callCount(4);
```

Comment:
Why did this change? I'm not sure why this would behave differently.

Reply:
I agree, no idea this could have been carried over from a prev ticket that didn't pick it up

Reply:
Actually, no it's because it's redirecting to the view app page now which has one less interaction with logger. Before it was redirecting to the app overview page and there were three calls to `getApplicationList` which is what accounted for the extra interaction with logger.

Reply:
Aha! you are correct of course. This is why we should be handling this stuff in stubbable methods so that we're only testing update and not also the one it redirects to.